ci/ansible/roles/pulp/tasks/main.yaml

```diff
@@ -164,16 +164,6 @@
       priority: 50
   when: pulp_build == "beta" or pulp_build == "stable"
 
-- name: Check fips state
```

Comment:
This might still be useful in the future, so might be worth leaving in.

Reply:
I understood that no matter what box we are installing pulp packages should have priority. I can adjust it.

Reply:
It was more about the check fips state task itself, since it registers a variable to do it.  But not a big deal either way, since currently the priority stuff is the only one that uses it.